src/inputs/ChipSelectField.stories.tsx

```diff
@@ -18,11 +19,12 @@ export function Example() {
     { id: "ts:1", code: TaskStatus.NotStarted, name: "Not Started" },
     { id: "ts:2", code: TaskStatus.InProgress, name: "In Progress" },
     { id: "ts:3", code: TaskStatus.Complete, name: "Complete" },
-    { id: "ts:5", code: TaskStatus.OnHold, name: "On Hold ".repeat(10) },
-    { id: "ts:6", code: TaskStatus.Delayed, name: "Delayed" },
+    { id: "ts:4", code: TaskStatus.OnHold, name: "On Hold ".repeat(10) },
+    { id: "ts:5", code: TaskStatus.Delayed, name: "Delayed" },
```

Comment:
Ahh was the Virtual item number change for this?

Reply:
Yes! You found it. Good 👁️